shared/mputil/benchmark_test.go

```diff
@@ -0,0 +1,51 @@
+package mputil_test
+
+import (
+	"crypto/rand"
+	"crypto/sha256"
+	"sync"
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/shared/mputil"
+)
+
+var input [][]byte
+
+func init() {
+	input = make([][]byte, 65536)
```

Comment:
What is the 65536 magic number? Can it be a const?

Reply:
It's just a value picked for the benchmarking but yeah will make it (and other values) constants.